pages/en/cookbook/grafting.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Grafting on Existing Subgraphs
```

Comment:
Maybe we can change the title to: "Replacing a contract address and keep history with Grafting"

Reply:
I think my reservation about that title suggestion is that I don't think Grafting is the best way to do that (vs. adding a new data source with the new address). Grafting is a means of quickly fixing (vs. a preferred upgrade pattern)

Reply:
ok, just changed it